### PR TITLE
change components name

### PR DESCRIPTION
**Pull request by catbelljune**
plz check components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import "./App.css";
-import MainPage from "./routers";
+import Main from "./routers"
```
Comment (pentagram5) [inline src/App.js:2]: good change!

File: src/App.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 import scrollbar from "smooth-scrollbar";
 
 scrollbar.initAll();
 
 function App() {
   return (
     <>
-      <MainPage />
+      <Main />
```
Comment (pentagram5) [inline src/App.js:10]: looks good to me!

**Review by pentagram5 — CHANGES_REQUESTED** [review]
plz change more components name

**Review by catbelljune — COMMENTED** [review]
change_more_name

**Review by pentagram5 — APPROVED** [review]
all good!

**Pull request merged**